app/src/main/java/com/mertceyhan/bitcoinmarket/features/market/domain/model/MarketInformation.kt

```diff
@@ -15,4 +15,6 @@ data class MarketInformation(
     val aboutChart: String,
     val timespan: MarketInformationTimespan,
     val chartEntries: List<Entry>
-)
+) {
+    val isChangeRatePositive: Boolean = changeStatus == MarketInformationChangeStatus.POSITIVE
```

Comment:
Can we put this into `BitcoinWidgetUiState` class?

Reply:
What is the reason behind that? I just wondered. What I want is keep as a attribute to access easily and don't want to do this in UI part.

Reply:
`MarketInformation` is a model, right?  with this `isChangeRatePositive`  variable, we're adding a logic inside a model, and also it's hard to test it cause it's not a function. I think a model class just should keep the data, nothing else. Is it reasonable for you?